ni/github.py

```diff
@@ -59,7 +63,7 @@
 """
 
 
-GITHUB_EMAIL = 'noreply@github.com'.lower()  # Normalized for easy comparisons.
+GITHUB_EMAIL = 'noreply@github.com'  # Normalized for easy comparisons.
```

Comment:
This was on purpose in case someone edited the email address and didn't remember to lower-case it. If this is going to be removed then please update the comment to say that it's normalized to lower-case.

Reply:
Aha!